README.md

```diff
@@ -1,11 +1,12 @@
 # SSP Operator
-Operator that manages Scheduling, Scale and Performance addons for [KubeVirt](https://kubevirt.io)
+Operator that installs resources for [KubeVirt](https://kubevirt.io)
 
 ## Functionality
 
 The operator deploys and manages resources needed by these four components:
 
-- [Template Validator](https://github.com/kubevirt/ssp-operator/tree/master/internal/template-validator) (You can read more about it here: [Template Validator old repository](https://github.com/kubevirt/kubevirt-template-validator))
+- [Common Instancetypes and Preferences Bundle](https://github.com/kubevirt/common-instancetypes/)
+- [Template Validator](https://github.com/kubevirt/ssp-operator/tree/master/internal/template-validator)
```

Comment:
Move this below common templates bundle?

Reply:
done